### DISCUSSION TITLE
About mingw, mingw-winlibs and mingw-nuwen

### DISCUSSION THREAD

**Discussion by Miqueas** [ScoopInstaller/Main#5688] [Q&A] [answered]
I'm just curious about the difference between these. For me they're the same but made by different people. Should I use one of these instead of GCC?

**Comment by chawyehsu** [comment, marked as answer]
They are just different mingw distros having opinionated differences derived by their distributors. It's hard to tell what particular differences between them, like it's hard to tell what particular differences between linux distros. You may need to investigate by yourself, the same way you try different linux distros to choose one you prefer.

**Reply by Miqueas** [reply]
Oooh I see, that's a really good answer tho, thanks!

**Reply by Miqueas** [reply]
Basically, it's the same base and the differences resides most likely in the philosophy
